sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2024-2025.bpmn

```diff
@@ -828,29 +822,63 @@
       <bpmn:incoming>Flow_1tcyqyg</bpmn:incoming>
       <bpmn:outgoing>Flow_15xhu93</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
+    <bpmn:sequenceFlow id="Flow_1tcyqyg" sourceRef="StartEvent_1" targetRef="Event_0rpww4j" />
+    <bpmn:sequenceFlow id="Flow_15xhu93" sourceRef="Event_0rpww4j" targetRef="Activity_1vwijyc" />
+    <bpmn:parallelGateway id="Gateway_08rr1xe">
+      <bpmn:incoming>Flow_1idr6ms</bpmn:incoming>
+      <bpmn:incoming>Flow_0n8rikf</bpmn:incoming>
+      <bpmn:outgoing>Flow_1gf8xyd</bpmn:outgoing>
+    </bpmn:parallelGateway>
+    <bpmn:sequenceFlow id="Flow_037lfnr" sourceRef="Gateway_0wuxqll" targetRef="Event_1mkiqso" />
+    <bpmn:sequenceFlow id="Flow_1vb0nqd" sourceRef="Gateway_0wuxqll" targetRef="Event_0asamuq" />
+    <bpmn:sequenceFlow id="Flow_0n8rikf" sourceRef="Event_0asamuq" targetRef="Gateway_08rr1xe" />
+    <bpmn:sequenceFlow id="Flow_1tkgnk6" sourceRef="Activity_1vwijyc" targetRef="Gateway_0wuxqll" />
+    <bpmn:parallelGateway id="Gateway_0wuxqll">
+      <bpmn:incoming>Flow_1tkgnk6</bpmn:incoming>
+      <bpmn:outgoing>Flow_037lfnr</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1vb0nqd</bpmn:outgoing>
+    </bpmn:parallelGateway>
+    <bpmn:scriptTask id="Activity_1vwijyc" name="Define Dependents">
+      <bpmn:extensionElements>
+        <zeebe:script expression="=// Consider only dependents who were born on or before the study end date.&#10;inputDataDependents[date(item.dateOfBirth) &#60;= date(offeringStudyEndDate)]" resultVariable="calculatedDataDependants" />
+        <zeebe:ioMapping>
+          <zeebe:input source="=//TODO: Student appeal variable needs to be considered here.&#10;if studentDataDependants = null &#10;then []&#10;else studentDataDependants" target="inputDataDependents" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_15xhu93</bpmn:incoming>
+      <bpmn:outgoing>Flow_1tkgnk6</bpmn:outgoing>
+    </bpmn:scriptTask>
+    <bpmn:intermediateThrowEvent id="Event_0asamuq" name="Calculate Financial Information">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if (is defined(appealsStudentFinancialInformationAppealData) and appealsStudentFinancialInformationAppealData != null)&#10;  then&#10;  appealsStudentFinancialInformationAppealData.taxReturnIncome &#10;else&#10;  studentDataTaxReturnIncome" target="calculatedDataTaxReturnIncome" />
+          <zeebe:output source="=if (is defined(appealsStudentFinancialInformationAppealData) and appealsStudentFinancialInformationAppealData != null)&#10;  then appealsStudentFinancialInformationAppealData.daycareCosts12YearsOrOver &#10;else&#10;  studentDataDaycareCosts12YearsOrOver" target="calculatedDataDaycareCosts12YearsOrOver" />
```

Comment:
Since we started using the `is defined` the [FEEL Null check](https://docs.camunda.io/docs/next/components/modeler/feel/language-guide/feel-boolean-expressions/#null-check) was probably improved and we no longer need the `is defined`.
Can you please check if the below code works when the variable is not present?

```
if (appealsStudentFinancialInformationAppealData != null) then
  appealsStudentFinancialInformationAppealData.taxReturnIncome 
else
  studentDataTaxReturnIncome
```

There is a better operator to execute a coalesce-like comparison ([get or else](if (appealsStudentFinancialInformationAppealData != null) then
  appealsStudentFinancialInformationAppealData.taxReturnIncome 
else
  studentDataTaxReturnIncome)) but apparently it has an open issue right now: https://github.com/camunda/camunda-modeler/issues/3983

Reply:
The FEEL null check works. Thanks.